stack/stack.toml

```diff
@@ -0,0 +1,366 @@
+id = "io.buildpacks.stacks.noble"
+homepage = "https://github.com/paketo-buildpacks/noble-full-stack"
+maintainer = "Paketo Buildpacks"
+
+platforms = ["linux/amd64", "linux/arm64"]
+
+[build]
+  description = "ubuntu:noble with compilers and common libraries and utilities"
+  dockerfile = "./build.Dockerfile"
+  gid = 1000
+  shell = "/bin/bash"
+  uid = 1001
+
+  [build.args]
+    architecture = "amd64"
+
+    sources = """
+Types: deb
+URIs: http://archive.ubuntu.com/ubuntu/
+Suites: noble noble-updates
+Components: main universe multiverse
+Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg
+
+Types: deb
+URIs: http://security.ubuntu.com/ubuntu/
+Suites: noble-security
+Components: main universe multiverse
+Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg
+"""
+
+    packages = """\
```

Comment:
compared this with the Jammy full packages, and found a few very minor differences (ex. `gnupg1-curl` is missing on the build image). I was wondering how you determined which packages to include here, since a number of them have slightly diff names?

Reply:
couple of packages (like `libsqlite0`) are no longer present in noble repositories, so we had to remove them from the list.

`gnupg1-curl` [is virtual](https://packages.ubuntu.com/noble/gnupg1-curl) and backed up by a single package `gnupg1` (which is already in the list) so it doesn't really make sense to keep it